### DISCUSSION TITLE
Non ASCII output is garbled

### DISCUSSION THREAD

**Discussion by sergeevabc** [ImageMagick/ImageMagick#3962] [Development]
Windows 7, ImageMagick 7, [орлы.png](https://user-images.githubusercontent.com/373718/126172617-257f61cf-500b-48b2-ae32-54c4a78459a6.png)

```
$ magick.exe -verbose орлы.png орлы.webp
РѕСЂР»С‹.png PNG 162x192 162x192+0+0 8-bit Gray 11768B 0.000u 0:00.024              
РѕСЂР»С‹.png=>РѕСЂР»С‹.webp PNG 162x192 162x192+0+0 8-bit sRGB 6428B 0.031u 0:00.091
^^^^^^^^
```

Tried to set various chcp 866, 1251, 65001 with no success, not sure what to try next.
Your colleagues from VIPS project, namely @jcupitt, [managed][1] to fix that, so it is possible. What about ImageMagick?

[1]: https://github.com/libvips/libvips/issues/294

**Comment by snibgo** [comment]
The forum host has renamed your uploaded .png file. But the problem can be demonstrated without that:
```
f:\web\im>chcp 65001
Active code page: 65001

f:\web\im>%IMG701010%magick -verbose rose: орлы.png
PNM:ROSE=>ROSE PNM 70x46 70x46+0+0 8-bit sRGB 9673B 0.000u 0:00.010
rose:=>ROSE PNM 70x46 70x46+0+0 8-bit sRGB 9673B 0.016u 0:00.043
rose:=>ÐPNM 70x46 70x46+0+0 8-bit sRGB 9673B 0.000u 0:00.165
```
Tested on IM v7.0.10-45 on Windows 8.1.

**Reply by sergeevabc** [reply]
Does this mean that you have confirmed the issue and the question should be reclassified as an issue?

**Comment by snibgo** [comment]
I confirm that there is an issue. With my command, IM creates a file with the correct name, but "-verbose" does not show the correct name. I would call this a bug.

A developer would need to look at this.

I think there is no need to open a thread on the issues forum https://github.com/ImageMagick/ImageMagick/issues

**Comment by sergeevabc** [comment]
Dear @urban-warrior and @dlemstra, could you be so kind to look into this?
Your colleague @snibgo was kind enough to confirm the issue, but I am not sure what to do next to make it resolved more quickly.

**Reply by dlemstra** [reply]
We will put this on or list of things to do but I have no idea when we will have time for this. Is this a Windows only issue?
